### ISSUE TITLE
[HelpDesk] xcode-11.1 need 'sudo xcode-select -s /Applications/Xcode.app/Contents/Developer'

### ISSUE THREAD

**Issue by halx99** [cocos2d/cocos2d-x-3rd-party-libs-src#181]
to make sure build.sh works well with xcode, need run follow command:
```sudo xcode-select -s /Applications/Xcode.app/Contents/Developer```

**Comment by minggo** [comment]
Yep, i met the issue too. Did you know if there is any side effect if the Xcode version is not 11.1+?

**Comment by halx99** [comment]
I have not test, I just upgrade our xode to 11.1, then requires run the above command for build.sh works well.
